### PR TITLE
build rules to create archives for android studio

### PR DESCRIPTION
**Pull request by loosebazooka**

### CODE REVIEW

**Comment by loosebazooka** [comment]
@patflynn

File: scripts/android-studio-build.gradle [hunk 1]
```diff
@@ -0,0 +1,27 @@
+task buildAndroidStudioRelease(type: Zip) {
+    archiveName 'androidStudioRelease.zip'
```
Comment (patflynn) [inline scripts/android-studio-build.gradle]: can we include our version number in the release zip name?

Comment (loosebazooka) [inline scripts/android-studio-build.gradle]: well really this is a zip for us to transfer it to their codebase, versions are still in the jars BUT, yes I can do that.

Comment (patflynn) [inline scripts/android-studio-build.gradle]: Thanks!!

File: scripts/android-studio-build.gradle [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    destinationDir = buildDir
+    def pluginDirs = [":google-cloud-tools-plugin", ":google-account-plugin"]
```
Comment (patflynn) [inline scripts/android-studio-build.gradle]: I'm noticing when I try out this script that the google-cloud-tools folder has the plugin sources.jar but not the actual code.jar.  google-account-plugin looks good though.

One other thing is that common-lib is missing the sources jar, and most of the lib jars don't have srcjars. Not sure if AS want that.

**Comment by loosebazooka** [comment]
oh yeah I mucked up the task dependencies, lemme check.

**Comment by loosebazooka** [comment]
This is a particularly gnarly looking build script, I'm sorry. @patflynn

File: scripts/android-studio-build.gradle [hunk 3]
```diff
@@ -0,0 +1,140 @@
+def jarLicenseMap = [
+        "GoogleFeedback.jar" : "apache2",
```
Comment (patflynn) [inline scripts/android-studio-build.gradle]: so there's no way to query this from maven central?

File: scripts/android-studio-build.gradle [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+            // includes jars
+            project.tasks.jar.outputs.files.each { outputFile ->
+                from (outputFile) {
+                    into "${project.intellij.pluginName}/lib"
+                }
+            }
+            // include sources jar
+            project.tasks.jarSources.outputs.files.each { outputFile ->
+                from (outputFile) {
+                    into "${project.intellij.pluginName}/lib/src"
```
Comment (patflynn) [inline scripts/android-studio-build.gradle]: typically don't src jars just get included along side the binaries?

**Pull request closed without merging**